InnerEye/ML/reports/classification_report.py

```diff
@@ -333,6 +334,8 @@ def plot_image_for_subject(subject_id: str,
     :param model_output: The predicted value for this image
     """
     print_header("", level=4)
+    if header:
+        print_header(header, level=4)
```

Comment:
Wouldn't it make more sense to have `print_header(header or "", level=4)`? At the moment, you'd get 2 prints?

Reply:
The empty string is just spacing to format it nicely.